include/transport/time_queue.h

```diff
@@ -333,7 +360,7 @@ namespace qtransport {
                 auto& [bucket_index, value_index, expiry_tick] = _queue.at(_queue_index++);
                 auto& bucket = _buckets.at(bucket_index);
```

Comment:
Old code I know, but just thinking these should probably be `const auto&`, not sure why they weren't before. Mea culpa.

Reply:
Updated.